src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/tests/TypeHeader.unit.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderWithStoreAndRouter } from '~/platform/testing/unit/react-testing-library-helpers';
+import TypeHeader from '../TypeHeader';
+import { formatHeader } from '../DetailsVA.util';
+
+describe('TypeHeader component', () => {
+  it('should create data-cy=va-appointment-details-header', async () => {
+    const appointment = {
+      vaos: {
+        isPastAppointment: true,
+      },
+    };
+    const header = formatHeader(appointment);
+    const wrapper = renderWithStoreAndRouter(
+      <TypeHeader>{header}</TypeHeader>,
+      {},
+    );
+
+    expect(
+      wrapper.getByText('VA appointment', {
+        exact: true,
+        selector: 'h2',
+      }),
+    ).to.have.attribute('data-cy', 'va-appointment-details-header');
+  });
+
+  it('should create data-cy=va-video-appointment-details-header', async () => {
+    const appointment = {
+      vaos: {
+        isPastAppointment: true,
+      },
+    };
+    const header = formatHeader(appointment);
+    const wrapper = renderWithStoreAndRouter(
+      <TypeHeader isVideo>{header}</TypeHeader>,
+      {},
+    );
+
+    expect(
+      wrapper.getByText('VA appointment', {
+        exact: true,
+        selector: 'h2',
+      }),
+    ).to.have.attribute('data-cy', 'va-video-appointment-details-header');
+  });
+
+  it('should create data-cy=community-care-appointment-details-header', async () => {
```

Comment:
The only thing I would consider changing would be to adjust the test case titles to human instead of the values of the attributes. 

This: `it('should create Community Care appointment details header', async () => {`

Instead of: `it('should create data-cy=community-care-appointment-details-header', async () => {`

Reply:
fixed